README.md

```diff
@@ -1,11 +1,67 @@
-# mamba.np 
+# mamba.np
+
 `mamba.np` is a pure NumPy implementation of Mamba.
 
 <p align="center">
   <img src="assets/mamba.jpeg" width="300" alt="mamba.np">
 </p>
 
+## Prerequisites
```

Comment:
Maybe move these sections into a new INSTALL.md file which a reference link from the usage section.